rest-api/offline/public_metrics_export.py

```diff
@@ -0,0 +1,204 @@
+import clock
+import collections
+
+from sqlalchemy import text
+from dao import database_factory
+from dao.database_utils import replace_years_old
+from dao.hpo_dao import HPODao
+from participant_enums import EnrollmentStatus, OrderStatus, PhysicalMeasurementsStatus
+from participant_enums import Race, QuestionnaireStatus
+from participant_enums import TEST_EMAIL_PATTERN, TEST_HPO_NAME
+
+
+def _questionnaire_metric(name, col):
+  """Returns a metrics SQL aggregation tuple for the given key/column."""
+  return _SqlAggregation(
+      name,
+      """
+      SELECT {col}, COUNT(*)
+      FROM participant_summary
+      WHERE {filter_test_sql}
+      GROUP BY 1;
+      """.format(col=col, filter_test_sql=_FILTER_TEST_SQL),
+      lambda v: QuestionnaireStatus.lookup_by_number(v).name,
+      None
+  )
+
+_FILTER_TEST_SQL = """
```

Comment:
We may also want to filter out participants who have withdrawn (participant_summary.withdrawal_status = :no_use)

@jmandel let us know if that sounds right.

Reply:
I added this to the filter.